src/Drupal/Commands/config/ConfigCommands.php

```diff
@@ -196,6 +197,86 @@ public function delete($config_name, $key = null)
     }
 
     /**
+     * Display status of configuration (differences between the filesystem configuration and database configuration).
+     *
+     * @command config:status
+     * @interact-config-label
+     * @param string $label A config directory label (i.e. a key in \$config_directories array in settings.php).
+     */
+    public function status($label = NULL)
```

Comment:
$label needs an @param declaration in doxygen

Reply:
There's already a `@param string $label ...` above this. Am I missing something?

Reply:
oops, my mistake. @param usually comes right below @command.